features/git-ship/current_branch/on_feature_branch/without_open_changes/without_remote_origin.feature

```diff
@@ -0,0 +1,32 @@
+Feature: git ship: shipping the current feature branch without a remote origin
+
+  As a developer having finished a feature and on repo without a remote origin
+  I want to be able to ship it safely in one easy step
+  So that I can quickly move on to the next feature and remain productive.
+
+
+  Background:
+    Given I have a feature branch named "feature"
+    And my repo does not have a remote origin
+    And the following commit exists in my repository
+      | BRANCH  | LOCATION | FILE NAME    | FILE CONTENT    |
+      | feature | local    | feature_file | feature content |
+    And I am on the "feature" branch
+    When I run `git ship -m "feature done"`
+
+
+  Scenario: result
+    Then it runs the Git commands
+      | BRANCH  | COMMAND                            |
+      | feature | git checkout main                  |
```

Comment:
Can you see if you can save `git-ship` from performing this extraneous checkout? Same optimization for `git-hack` can be made as well.